icu4j/APIChangeReport.html

```diff
@@ -4,260 +4,231 @@
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
 <!-- © 2021 and later: Unicode, Inc. and others. -->
 <!-- License & terms of use: http://www.unicode.org/copyright.html -->
-<title>ICU4J API Comparison: ICU4J 68.1 with ICU4J 69.1</title>
+<title>ICU4J API Comparison: ICU4J 69.1 with ICU4J 70.0.1</title>
```

Comment:
I think we should set the version to 70.1 at this point, so we don't need to update it for 70.1 GA.

Reply:
The version issue - you should wat this -> https://github.com/unicode-org/icu/pull/1873

Reply:
Thx. I've just added the ICU4C API Change report to this PR, squashing into a single commit.

I think PR 1873 will update the version to 70.1 - I can redo these API reports after that.

Reply:
Erik merged PR 1873 yesterday. Please update.